client/src/utils/savePacket.ts

```diff
@@ -0,0 +1,50 @@
+import { IMission, IDataPoint } from "./entities";
+import { IAprsTelemetryPacket } from "./TelemetryTypes";
+import api from "../services/api";
+
+export const saveAprsPacket = async (mission: IMission , packet: IAprsTelemetryPacket, packetNumber: number) => {
+    console.log('in save packet:', packet)
+    let missionId: string = '';
+
+    if (!mission._id) {
+        return 'No mission Id attached';
+    } else {
+        missionId = mission._id;
+    } 
+
+    const longPayload: IDataPoint = {
+        TimeSinceLaunch: Number(packet.Parsed.timeStampUnix),
+        Value: (packet.Parsed.longitude) as number,
+        PacketNumber: packetNumber,
+        FieldId: '64e931e35a8391d54367af01',
```

Comment:
What are these hardcoded values for `FieldId`?

Reply:
This was just easier to connect data to the fields durinig compitition. I will kill them before we merge